eZ/Publish/Core/REST/Server/Service/RootService.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+/**
+ * File containing the resource service class.
+ *
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ *
+ * @version //autogentag//
+ */
+namespace eZ\Publish\Core\REST\Server\Service;
+
+use eZ\Publish\Core\REST\Common\Values;
+use eZ\Publish\Core\REST\Common\Values\Root;
+use Symfony\Component\ExpressionLanguage\ExpressionLanguage;
+use Symfony\Component\Routing\RouterInterface;
+
+class RootService
```

Comment:
Yeah, `buildRootResource` would do the job constructor does and return it immediatelly, no need for `getRoot` then.